### DISCUSSION TITLE
Sound Problems

### DISCUSSION THREAD

**Discussion by Hyper-Snoc** [rocksdanister/lively#977] [Q&A] [answered]
[lively_log_20220102_093733.zip](https://github.com/rocksdanister/lively/files/7799315/lively_log_20220102_093733.zip)
I'm trying to listen to music from a stream I'm using as a wallpaper but when I use other programs the sound stops. Is there something I'm missing?

**Comment by rocksdanister** [comment, marked as answer]
Settings -> Audio - > Uncheck "Play audio only when desktop is focused."
